libs/react/src/lib/icons/arrowUpRightSquare.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { createComponent } from '@lit/react'
+import { IconSquareArrowTopRight } from '@sebgroup/green-core/components/icon/icons/square-arrow-top-right'
+import { getScopedTagName } from '@sebgroup/green-core/scoping'
+
+export const ArrowUpRightSquare = createComponent({
```

Comment:
Name should be `SquareArrowTopRight` to match with icon lib naming.

Reply:
Updated file